spec/features/admin/allocations_spec.rb

```diff
@@ -67,8 +67,8 @@
 
       before { click_link 'Novo(a) Alocação' }
 
-      it 'should not render users allocated' do
-        expect(page).not_to have_text(user.name)
+      it 'should render users allocated' do
```

Comment:
I know you're just using the wording from the previous spec, but let's start migrating on to the correct spec wording and remove `should` (unnecessary when using `expect`).
```suggestion
      it 'renders the allocated users' do
```
See more: https://www.betterspecs.org/#expect

Reply:
Ok.